### PR TITLE
add viz & metrics

### PR DESCRIPTION
**Pull request by dfeeney31**

### CODE REVIEW

File: Python/PerformanceTest_Ski_ExtractVarsLoadSol.py [hunk 1]
```diff
@@ -20,6 +20,14 @@
 from tkinter import messagebox
 
 
+fPath = 'C:\\Users\\daniel.feeney\\Boa Technology Inc\\PFL Team - General\\Testing Segments\\Snow Performance\\Alpine_CuffOnSnow_Apr2022\\XSENSORdata\\'
```
Comment (EricHonertBOA) [inline Python/PerformanceTest_Ski_ExtractVarsLoadSol.py:23]: May want to delete this additional line

File: Python/Visualization_SkiTurnsTS.py [hunk 2]
```diff
@@ -20,13 +20,14 @@
 # select files
 
 fPath = 'C:\\Users\\daniel.feeney\\Boa Technology Inc\\PFL Team - General\\Testing Segments\\Snow Performance\\Alpine_CuffOnSnow_Apr2022\\XSENSORdata\\'
+fPath = 'C:\\Users\\daniel.feeney\\Boa Technology Inc\\PFL Team - General\\Testing Segments\\Snow Performance\\SkiValidation_Dec2022\OnSnowPilot\\'
 fileExt = r".txt"
 entries = [fName for fName in os.listdir(fPath) if fName.endswith(fileExt)]
 entries = os.listdir(fPath)
```
Comment (EricHonertBOA) [inline Python/Visualization_SkiTurnsTS.py:26]: This line is redundant and potentially problematic as it re-pulls all files in the directory. Please delete

File: Python/Visualization_SnowboardTurnsTS.py [hunk 3]
```diff
@@ -20,6 +20,7 @@
 # select files
 
 fPath = 'C:\\Users\\daniel.feeney\\Boa Technology Inc\\PFL Team - General\\Testing Segments\\Snow Performance\\SB_2DialTakeDown_Mar2022\\Forces\\'
+#fPath = 'C:\\Users\\daniel.feeney\\iCloudDrive\\iCloud~de~novel~loadsols\\'
 fileExt = r".txt"
 entries = [fName for fName in os.listdir(fPath) if fName.endswith(fileExt)]
 entries = os.listdir(fPath)
```
Comment (EricHonertBOA) [inline Python/Visualization_SnowboardTurnsTS.py:26]: This line is redundant and potentially problematic as it re-pulls all files in the directory. Please delete

File: Python/Visualization_SkiTurnsTS.py (hunk 2 above)
Comment (dfeeney31) [reply Python/Visualization_SkiTurnsTS.py:26]: these lines are for testing and can be deleted once it is closer to production. For now, the previous line works great so need to ensure both work

Comment (dfeeney31) [reply Python/Visualization_SkiTurnsTS.py:26]: it also only sets the variable temporarily so is not a performance issue for testing

File: Python/Visualization_SnowboardTurnsTS.py (hunk 3 above)
Comment (dfeeney31) [reply Python/Visualization_SnowboardTurnsTS.py:26]: same comments as above. will delete once we are further through development issues

**Review by EricHonertBOA — APPROVED** [review]

**Pull request merged**